contracts/interchain-token-service/src/event.rs

```diff
@@ -13,6 +13,26 @@ pub struct TrustedChainRemovedEvent {
     pub chain: String,
 }
 
+#[derive(Debug, PartialEq, Eq)]
+pub struct InterchainTokenDeployedEvent {
+    pub token_id: BytesN<32>,
+    pub token_address: Address,
+    pub minter: Option<Address>,
+    pub name: String,
+    pub symbol: String,
+    pub decimals: u32,
+}
+
+#[derive(Debug, PartialEq, Eq)]
+pub struct InterchainTokenDeploymentStartedEvent {
+    pub token_id: BytesN<32>,
+    pub name: String,
+    pub symbol: String,
+    pub decimals: u32,
+    pub minter: Option<Address>,
+    pub destination_chain: String,
```

Comment:
```suggestion
    pub token_id: BytesN<32>,
    pub token_address: Address,
    pub destination_chain: String,
    pub name: String,
    pub symbol: String,
    pub decimals: u32,
    pub minter: Option<Address>,
```
add token_address for indexing, and move destination chain above

Reply:
it's fine to deviate a bit from the EVM event since it's extending it